Usiqee/Module/Account/AccountSettingsVC.swift

```diff
@@ -49,10 +49,10 @@ class AccountSettingsVC: UIViewController {
     
     private func setupTranslation() {
         titleLabel.text = L10N.AccountSettings.title
-        legalNoticeLabel.text = L10N.AccountSettings.legalNotice
+        legalNoticeLabel.text = L10N.AccountSettings.legalNotice.uppercased()
         privacyButton.setTitle(L10N.AccountSettings.privacy, for: .normal)
         cguButton.setTitle(L10N.AccountSettings.cgu, for: .normal)
-        notificationsLabel.text = L10N.AccountSettings.notifications
+        notificationsLabel.text = L10N.AccountSettings.notifications.uppercased()
```

Comment:
a remettre sans le uppercased, c'est manageable depuis AppLanga mtn 👍

Reply:
j'ai peur que si il le change plus tard en minuscule, il demande pk

Reply:
ouais you're right, autant garder toute la gestion sur applanga au cas ou il change d'avis